classes/autotools.oeclass

```diff
@@ -48,7 +48,16 @@ autotools_configure[emit] = "do_configure"
 do_configure () {
     do_configure_autotools
 }
+AUTOCONFDIR = ""
+UPDATE_GNUCONF ?= "0"
+UPDATE_GNUCONF:TARGET_CPU_aarch64 = "1"
+
 do_configure_autotools () {
+    if [ -a "${UPDATE_GNUCONF}" != "0" ] ; then
```

Comment:
I think you have an "-a" to much in there.  But otherwise, yes, this is better :)